src/plugins/rel/rel.js

```diff
@@ -162,14 +598,29 @@
                 z: el.getAttribute( "data-z" ),
                 relX: el.getAttribute( "data-rel-x" ),
                 relY: el.getAttribute( "data-rel-y" ),
-                relZ: el.getAttribute( "data-rel-z" )
+                relZ: el.getAttribute( "data-rel-z" ),
+                rotateX: el.getAttribute( "data-rotate-x" ),
+                rotateY: el.getAttribute( "data-rotate-y" ),
+                rotateZ: el.getAttribute( "data-rotate-z" ),
+                relRotateX: el.getAttribute( "data-rel-rotate-x" ),
+                relRotateY: el.getAttribute( "data-rel-rotate-y" ),
+                relRotateZ: el.getAttribute( "data-rel-rotate-z" ),
+                relPosition: el.getAttribute( "data-rel-position" ),
+                rotateOrder: el.getAttribute( "data-rotate-order" ),
+                relRotateOrder: el.getAttribute( "data-rel-rotate-order" )
```

Comment:
I don't understand the last one. How could there be a different rotate order for relative vs absolute? I don't think this  last attribute is needed, or possible.

Reply:
As it's said in examples/3D-rotations/index.html, rotating order matters, relative rotating needs order too.  It's ok to use the global `data-rotate-order`, as we disallow mixing up `rotate-x/y/z` and `rel-rotate-x/y/z`, there's no need for two rotation-order in one slide.

Reply:
Rotate order matters, but there can't be two orders effective at the same time.

Also, I'm not going to approve the proposal to disallow mixing relative and absolute coordinates. It's not how rel is designed to work.

Reply:
Sorry for the late reply.
I've modified the plugin to allow mixing relative and absolute coordinates.